tests/unit/utils/test_time_formats.py

```diff
@@ -0,0 +1,68 @@
+from typing import List, Optional, Union
+
+from merlin.utils import convert_timestring
+
+
+def _list_all_equal(my_list: List) -> bool:
+    """Boolean check that all items in the list are the same"""
+    return all(item == my_list[0] for item in my_list)
+
+
+def check_all_equal(
+    time_strings: List[Union[str, int]], method: Optional[str] = "HMS"
+) -> bool:
+    """Check that time strings in a list convert to the same for the given converstion method."""
+    converted = [convert_timestring(ts) for ts in time_strings]
+    all_equal = _list_all_equal(converted)
+    return all_equal
+
+
+def check_convert_explicit(
+    time_string: str, expected_result: str, method: str = "HMS"
+) -> None:
+    """Check an explicit conversion against the expected result."""
+    converted = convert_timestring(time_string, method)
+    assert converted == expected_result
+
+
+def test_convert_explicit() -> None:
+    """Test some cases to make sure they are converted correctly."""
+    cases = [
+        ("01:00:00", "01:00:00", "HMS"),
+        ("01:00:00", "3600.0s", "FSD"),
+        ("1:00", "00:01:00", "HMS"),
+        ("1:00", "60.0s", "FSD"),
+        ("1", "00:00:01", "HMS"),
+        ("1", "1.0s", "FSD"),
+    ]
+    for time_string, expected_result, method in cases:
+        check_convert_explicit(time_string, expected_result, method)
+
+
+def test_convert_timestring_same() -> None:
```

Comment:
I'd refactor this and the next test, and also the previous. Rather than using data internal to the unit tests, I'd leverage either Pytest's Fixtures or Pytest's mark.parametrize, and also merge this and the next unit test, especially since they're both going through check_all_equal() to examine results, and just include the rubric with the test data.

The benefit is A compartmentalization, data is separated from test structure, which more resembles the runtime environment, and B if something fails out, a fixture or mark.parametrize decorator still runs against all the parameters independently, whereas internal data like this will fail dependently - test case 3 could've passed or failed, but we won't know because case 2 failed and stopped the test prematurely for all parameters, we'd have to fix test 2 and run again from the top to get results for tests beyond 2 and just hope no other tests break.

The argument against is: this is done and works :D

Let me know which you'd prefer if you'd want to explore more  - fixtures are generally meant to be recycled across different tests (and especially different related or dependent modules), marks are fixed to the test they decorate. I generally prefer Fixtures, but marks are very useful when you to limit the scope and a bit more readable since they're attached exactly where they're needed.